testing/todocheck_test.go

```diff
@@ -591,7 +591,7 @@ func TestGroovyTodos(t *testing.T) {
 	}
 }
 
-// Also tests Rust TODOs as rust uses the same comment syntax
+// Also tests Rust and Kotlin TODOs as they use the same comment syntax
```

Comment:
can you: 
 * include a file in `testing/scenarios/swift/` which is a kotlin source file & include todo err scenarios for it as well?
 * Could you do the same for Rust as it seems to have been missed?

Reply:
The changes requested have now been included @preslavmihaylov